### PR TITLE
fix: various untranslatable strings

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

I've done a round and noticed few strings that are not translatable (and thus not translated yet):
* Mostly `aria-label`s (for accessibility) added strings in `en/main.json` when necessary.
* In the "Pending Transactions" window, fixed empty texts for each tab which had a part of the string hardcoded.
* In the Arkeo Airdrop page, fixed the styling of one of the CTA buttons which was overflowing in certain languages.
* In the Trade page/component, the "Balance:" for each asset was hardcoded. 

Translations for the new strings will be added during the next Translation sync by the Globalization WS.

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk
Low. Mostly strings that are usually not visible to users.

## Testing
Besides the changes in the screenshots below there should be no visible change to end users and the aria labels will only be translated later, so basically test that the changes below work in all languages.

### Engineering
☝️ 

### Operations
☝️ 

### "Pending Transactions" window
Before: image
After: image

### Arkeo Airdrop page
Before: image
After: image

###  Trade page/component ("Balance:")
Before: image
After: image

### CODE REVIEW

File: src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx [hunk 1]
```diff
@@ -268,7 +268,9 @@ export const BorrowInput = ({
               onClick={onSubmit}
               isDisabled={Boolean(quoteErrorTranslation)}
             >
-              {quoteErrorTranslation ? translate(quoteErrorTranslation) : 'Borrow'}
+              {quoteErrorTranslation
+                ? translate(quoteErrorTranslation)
+                : translate('lending.borrow')}
```
Comment (gomesalexandre) [inline src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx:273]: 💜

File: src/components/Layout/Header/TxWindow/TxWindow.tsx [hunk 2]
```diff
@@ -54,9 +54,10 @@ const TxsByStatus: React.FC<TxsByStatusProps> = ({ txStatus, limit }) => {
   }, [limit, txIds])
 
   if (limitTxIds.length === 0) {
+    const translatedStatus = translate(`transactionRow.${txStatus.toLowerCase()}`)
```
Comment (gomesalexandre) [inline src/components/Layout/Header/TxWindow/TxWindow.tsx:57]: Translations are missing `TxStatus.SUCCESS` though it's not needed as success Txs shouldn't appear in the Tx window ✅

File: src/components/Layout/Header/NavBar/Notifications.tsx [hunk 3]
```diff
@@ -150,7 +152,7 @@ export const Notifications = memo(() => {
           disableAnalytics={disableAnalytics}
         >
           <NotificationFeed gapFromBell={10} placement='bottom-end'>
-            <IconButton aria-label='Open notifications'>
+            <IconButton aria-label={translate('navBar.openNotif')}>
```
Comment (gomesalexandre) [inline src/components/Layout/Header/NavBar/Notifications.tsx:155]: Very nitpicky but prefer camelCasing the translation keys when possible for grepping purposes i.e `Open Notifications` -> `openNotifications` but no need to tackle here

**Review by gomesalexandre — APPROVED** [review]
Absolute animal 🥇

File: src/components/Layout/Header/NavBar/Notifications.tsx (hunk 3 above)
Comment (firebomb1) [reply src/components/Layout/Header/NavBar/Notifications.tsx:155]: Fair enough, I was aiming for shorter but still understandable with camelCase :D

**Pull request merged**